src/styles/blocks/card.css

```diff
@@ -0,0 +1,107 @@
+body {
+  background-color: var(--midium-bg-color);
+}
+
+.card {
+  min-width: 200px;
+  display: flex;
+  padding: 32px 16px 16px;
+  flex-direction: column;
+  margin-left: 56px;
+  width: 200px;
+  font-family: "Roboto", sans-serif;
+  font-weight: 400;
+  font-size: 10px;
+
+  background-color: var(--light-bg-color);
+
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  --light-bg-color: #fff;
```

Comment:
Move CSS custom properties for colors (lines 21-29) to a separate variables.css file.